bids2openminds/mapping.py

```diff
@@ -0,0 +1,152 @@
+MAP_2_EXPERIMENTAL_APPROACHES={

```

Comment:
we wanted to directly use openminds python objects for the mappings. S can help with this